codenames-front-end/src/components/game/running/CNRunGameFrame/CNRunGameFrame.tsx

```diff
@@ -0,0 +1,70 @@
+import React, {FC, useEffect} from 'react';
+
+import css from './CNRunGameFrame.module.css'
+import {Status} from "../../../../models/CodeNames/Status";
```

Comment:
May be simplified (install SonarLint to handle it automatically)

Reply:
So I installed it, but it didn't offer anything better